.gitlab-ci.yml

```diff
@@ -72,6 +72,8 @@ build-wasm-peregrine:
     - cp -r * /build
     - /srtool/build build
     - subwasm meta --format=json+scale /out/peregrine_runtime.compact.wasm > /out/peregrine-metadata.json
+    - subwasm get -o peregrine-live.wasm wss://peregrine.kilt.io
+    - subwasm diff --no-color peregrine-live.wasm /out/peregrine_runtime.compact.wasm > /out/peregrine-diff.txt
```

Comment:
So the output is redirected to a file, and we don't expose it as an artifact. How can we use the result of the diff in this case?

Reply:
Good catch i thought everything in `out/` is an artifact

Reply:
I thought about that as well. I think printing it to the output is not nice since it's harder to copy/past/store. During the release process you will download the artifacts anyways, so you will have this output there as well.

I personally can't process the JSON. I'm not sure if there are tools that use the JSON but I'm not aware of any. That's why I choose the text output.

Reply:
might also make sense to use `tee` so that it's printed as well

Reply:
[f38059e](https://github.com/KILTprotocol/kilt-node/pull/549/commits/f38059e401ad5f0fa1c55ac78402ae81353c2f33)